src/Raven.Client/Documents/Session/Loaders/MultiLoaderWithInclude.cs

```diff
@@ -67,14 +68,59 @@ public ILoaderWithInclude<T> Include<TInclude>(Expression<Func<T, IEnumerable<st
             return Include(IncludesUtil.GetPrefixedIncludePath<TInclude>(path.ToPropertyPath(), _session.Conventions));
         }
 
+        /// <summary>
+        /// Includes a counter of the document to be loaded, by counter name
+        /// <param name="name">Name of the counter to include.</param>
+        /// </summary>
+        public ILoaderWithInclude<T> IncludeCounter(string name)
+        {
+            if (string.IsNullOrWhiteSpace(name))
+                throw new ArgumentNullException(nameof(name));
+
+            if (_counters == null)
+                _counters = new List<string>();
+
+            _counters.Add(name);
+            return this;
+        }
+
+        /// <summary>
+        /// Includes counters of the document to be loaded, by counters names
+        /// <param name="names">Names of the counters to include.</param>
+        /// </summary>
+        public ILoaderWithInclude<T> IncludeCounters(string[] names)
+        {
+            if (_counters == null)
+                _counters = new List<string>();
+
+            if (names == null)
+                throw new ArgumentNullException(nameof(names));
+
+            foreach (var name in names)
+            {
+                if (string.IsNullOrWhiteSpace(name))
+                    throw new InvalidOperationException("IncludeCounters(string[] names) : 'names' should not contain null or whitespace elements");
+                _counters.Add(name);
+            }
+            return this;
+        }
+
+        /// <summary>
+        /// Includes all the counters of the document to be loaded
+        /// </summary>
+        public ILoaderWithInclude<T> IncludeCounters()
+        {
+            return IncludeCounters(new string[0]);
```

Comment:
don't like this, will fail in funny ways. `IncludeCounters` and then `IncludeCounter` will only load the specific counter we wanted.

Reply:
`IncludeCounter(name)`
`IncludeCounters(names)`
`IncludeAllCounters`
?

Reply:
```